tools/rndf_visualizer/src/MapLanes.cc

```diff
@@ -972,25 +973,18 @@ void MapLanes::testDraw(bool with_trans, const ZonePerimeterList &zones, bool sv
     }
 #endif
 
-	svg::Color blueColor = svg::Color(svg::Color::Blue);
-	svg::Color redColor = svg::Color(svg::Color::Red);
-	svg::Color greenColor = svg::Color(svg::Color::Green);
 	svg::Color cyanColor = svg::Color(svg::Color::Cyan);
 	svg::Color darkGrayColor = svg::Color(64, 64, 64);
 
-	svg::Stroke lineStrokeBlue = svg::Stroke(4, blueColor);
-	svg::Stroke lineStrokeRed = svg::Stroke(4, redColor);
-	svg::Stroke lineStrokeGreen = svg::Stroke(4, greenColor);
-	svg::Stroke lineStrokeCyan = svg::Stroke(4, cyanColor);
-	svg::Stroke lineStrokeDarkGray = svg::Stroke(4, darkGrayColor);
-
-	svg::Fill greenFill = svg::Fill(svg::Color::Green);
-	svg::Fill redFill = svg::Fill(svg::Color::Red);
-	svg::Fill blueFill = svg::Fill(svg::Color::Blue);
-	svg::Fill fuchsiaFill = svg::Fill(svg::Color::Fuchsia);
-	svg::Fill orangeFill = svg::Fill(svg::Color::Orange);
-	svg::Fill magentaFill = svg::Fill(svg::Color::Magenta);
-	svg::Fill yellowFill =  svg::Fill(svg::Color::Yellow);
+	svg::Color colors[] = {svg::Color::Blue,
```

Comment:
Also very minor, could you add a couple more colors (around 15 maybe)?  In cities like the 10k is somehow fuzzy.